src/renderer/contexts/SchemaProvider.tsx

```diff
@@ -20,15 +24,21 @@ export function useSchema() {
 export function SchemaProvider({
   children,
   schema,
-}: PropsWithChildren<{ schema?: DatabaseSchemas }>) {
+  reloadSchema,
+}: PropsWithChildren<{ schema?: DatabaseSchemas; reloadSchema: () => void }>) {
   const { setting } = useDatabaseSetting();
   const [currentDatabase, setCurrentDatabase] = useState(
     setting?.config?.database
   );
 
   return (
     <SchemaContext.Provider
-      value={{ schema: schema, currentDatabase, setCurrentDatabase }}
+      value={{
+        schema,
+        currentDatabase,
+        setCurrentDatabase,
+        reloadSchema,
+      }}
     >
```

Comment:
This is also my fault. The good implementation should be moving it to useMemo.

https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=94&id=invisal_query-master&open=AYm8naW0_gI_6lDKJJ8P

We can do

```typescript
const providerValuesMemo = useMemo(() => ({
 schema, currentDatabase, setCurrentDatabase, reloadSchema
 }), [schema, currentDatabase, setCurrentDatabase, reloadSchema])
```

Then we can do

```typescript
<SchemaContext.Provider value={providerValuesMemo}>
```